### ISSUE TITLE
Merge custom DMXOPL mod into GS GENMIDI .wopl file

### ISSUE THREAD

**Issue by freq-mod** [sneakernets/DMXOPL#47]
See discussion at f54a1e9.
Instrument changes described, I might open a PR if needed.

**Comment by sneakernets** [comment]
Okay, does this one have the "Full Vibrato" and "Full Tremolo" disabled? That was causing splattering problems in Heretic and made some of the strings sound way off to me. Also, the Choir Aahs sounded very odd in Doom 2 D_DM2INT (Intermission music), but I don't know if that had been addressed. I'll have to take a look at this tonight. Thanks for all the help though, I appreciate it!!

**Comment by freq-mod** [comment]
OW CRAP THATS RIGHT
I made a new Choir AAHs:
![aaaa](https://user-images.githubusercontent.com/32672779/45645592-3d9db400-bac1-11e8-859d-ccdc2c52b0af.PNG)

It's from even newer revison that .wopl I shared with you. You shouldn't bother about that bank right now, I may polish it a bit. 

EDIT: you may want to up a modulator 1 value a little bit

**Comment by sneakernets** [comment]
I forget if I implemented these changes @freq-mod  so I'm gonna look into them ASAP, unless you have a updated version that I can take a look at that is in WOPL format so we can preserve as many settings as possible

**Comment by freq-mod** [comment]
No, you did not.
Although a shitton have changed since 2018 with regards to OPL3 sound design, if Furnace scene is anything to go by, so a lot can be even further improved or redone. For now grab this and have fun or something. 

[GENMIDI(GS).zip](https://github.com/user-attachments/files/21326717/GENMIDI.GS.zip)

**Comment by sneakernets** [comment]
Yeah, I like to think I was partially responsible for the OPL3 resurgence... 😁 

there's a lot of stuff that could use an _extreme_ overhaul in DMXOPL, some stuff hasn't aged too well, and Blood's midis wreck it pretty good

also WOW this is a lot of changes!

**Comment by freq-mod** [comment]
> Blood's midis wreck it pretty good

can you elaborate? what is wreckage and what is responsible for it?
